presenters/workspace_manager_presenter.py

```diff
@@ -47,6 +47,8 @@ def _load_workspace(self):
         #TODO specify workspace name on load
         #TODO what to do on fail?
         workspace_to_load = self._workspace_manger_view.get_workspace_to_load_path()
+        if not workspace_to_load:
```

Comment:
We should try and avoid fixing too many issues in a single PR. It is small so we can let it go here but please update the PR description to fix the new issue.